README.md

```diff
@@ -1,2 +1,6 @@
 # nlp-timeline-gen
-A tool for generating character timelines from books using nlp.
+
+Novels often contain many places, characters and organisations each with their own unique names. As the plot thickens,
+these can become difficult to follow.
+nlp-timeline-gen aims to make this easier by generating a timeline of where people are at what times using 
+[state-of-the-art natural language processing](https://spacy.io/).
```

Comment:
You could add a line below (maybe with a divider before) mentioning that this is for the Year 3 SWE Group Project at Imperial College London.

Reply:
Done.